test/index.js

```diff
@@ -441,13 +449,68 @@ describe('hyperlink', function() {
         t
       );
 
+      expect(spyTapCalls(t.push), 'with set semantics to satisfy', [
```

Comment:
As I mentioned in https://gitter.im/unexpectedjs/unexpected?at=5d3475f2ec5abe7bbc227eab you should be able to use `to have calls satisfying here`.